app/javascript/components/map/map.js

```diff
@@ -22,6 +23,19 @@ const mapStateToProps = ({ location, datasets, geostore, latest }) => ({
 });
 
 class MapContainer extends PureComponent {
+  static propTypes = {
+    basemap: PropTypes.object,
+    mapOptions: PropTypes.object,
+    setLandsatBasemap: PropTypes.func
+  };
+
+  componentDidUpdate({ mapOptions: { prevZoom } }) {
```

Comment:
I feel like this is an operation of the base maps component and you are already getting the mapOptions there so this could go there.

Reply:
not really cause we need this always, even if the basemaps component is unmounted

Reply:
Ok.